modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceConfigurationUpdateHandler.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.deployment;
+
+import jakarta.annotation.Resource;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.LockModeType;
+import jakarta.transaction.Transactional;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.database.serviceconfiguration.update.ServiceConfigurationUpdateRequest;
+import org.eclipse.xpanse.modules.database.serviceconfiguration.update.ServiceConfigurationUpdateStorage;
+import org.eclipse.xpanse.modules.models.serviceconfiguration.enums.ServiceConfigurationStatus;
+import org.hibernate.exception.LockTimeoutException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * Service configuration more detailed request handler.
+ */
+@Slf4j
+@Component
+public class ServiceConfigurationUpdateHandler {
+
+    public static final String JAVAX_PERSISTENCE_LOCK_TIMEOUT = "javax.persistence.lock.timeout";
+
+    @Value("${javax.persistence.lock.timeout:2000}")
+    private int lockTimeout;
+
+    @Resource
+    private EntityManager entityManager;
+
+    @Resource
+    private ServiceConfigurationUpdateStorage serviceConfigurationUpdateStorage;
+
+    /**
+     * Update service configuration update request.
+     */
+    @Transactional
+    public ServiceConfigurationUpdateRequest
+            updateServiceConfigurationUpdateRequest(UUID id, String resourceName) {
+        try {
+            Map<String, Object> properties = new HashMap<>();
+            properties.put(JAVAX_PERSISTENCE_LOCK_TIMEOUT, lockTimeout);
+            ServiceConfigurationUpdateRequest request = entityManager
+                    .find(ServiceConfigurationUpdateRequest.class, id,
+                            LockModeType.PESSIMISTIC_WRITE, properties);
+            if (Objects.isNull(request.getResourceName())) {
```

Comment:
When the request is null, the code `request.getResourceName()` will throw `NullPointerException`.  Please check it and output the null log.